system_tests/language.py

```diff
@@ -88,7 +88,7 @@ def _check_analyze_entities_result(self, entities):
         self.assertEqual(entity2.metadata, {})
         # Verify entity 3.
         self.assertEqual(entity3.name, self.NAME3)
-        self.assertEqual(entity3.entity_type, EntityType.WORK_OF_ART)
+        self.assertEqual(entity3.entity_type, EntityType.EVENT)
```

Comment:
Hilariously, this used to be `EVENT` and it flaked its way to `WORK_OF_ART`, which we had to address in #2587

Maybe the better fix would be:

```python
choices = (EntityType.EVENT, EntityType.WORK_OF_ART)
self.assertIn(entity3.entity_type, choices)
```